graas/api.py

```diff
@@ -12,14 +12,14 @@ class GraasApi(object):
 
     app = Klein()
     static_path = pkg_resources.resource_filename(__name__, "static")
+    p = FilePath(static_path).child("index.html")
```

Comment:
Can we maybe name this `index_path`, or something more descriptive to what it is?